### PR TITLE
feat: migrate to ubuntu

### PR DESCRIPTION
**Pull request by innovaker**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Lightweight Docker images for [ZMK][zmk].
 
 For _building_ [ZMK][zmk] firmware with CI.
 
-- FROM: **[debian][debian]:stable-slim**
+- FROM: **[ubuntu][ubuntu]**
```
Comment (petejohanson) [inline README.md:30]: Should we list `focal` here as well, for clarity?

**Review by petejohanson — COMMENTED** [review]
Just one question. Otherwise looks good.

File: README.md (hunk 1 above)
Comment (innovaker) [reply README.md:30]: Yeah, I wondered that also.  I figured it's one more thing to forget when we upgrade `ubuntu` in the future.  Thoughts?

Comment (innovaker) [reply README.md:30]: People will look in the `Dockerfile` if they want to know the specifics.

Comment (petejohanson) [reply README.md:30]: WFM.

**Review by petejohanson — APPROVED** [review]

**Pull request merged**